Makefile

```diff
@@ -152,7 +154,7 @@ defn: $(defn_files)
 	$(K_BIN)/kompile --debug --main-module ETHEREUM-SIMULATION --backend java \
 					--syntax-module ETHEREUM-SIMULATION $< --directory .build/java -I .build/java
 
-.build/haskell/driver-kompiled/definition.kore: $(haskell_files)
+.build/haskell/driver-kompiled/definition.kore: haskell-defn
```

Comment:
This should be `$(haskell_files)`, not `haskell-defn`. Having a phony target as a prerequisite means the phony target will be rebuilt too often.